### PR TITLE
First test of wall that get position x and y succesful

### PR DESCRIPTION
**Pull request by JhordanSoto**
Created an object wall and we made a test to walls got position x and y.

### CODE REVIEW

File: src/test/java/pacman/WallandDotsTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WallandDotsTest {
```
Comment (maryangela) [inline src/test/java/pacman/WallandDotsTest.java]: name of class

File: src/test/java/pacman/WallandDotsTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import static org.junit.Assert.*;
+
```
Comment (maryangela) [inline src/test/java/pacman/WallandDotsTest.java]: remove space blank

File: src/test/java/pacman/WallandDotsTest.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import org.junit.Test;
+
+public class WallandDotsTest {
+	@Test
+	public void testExist(){
+        int x=1;
```
Comment (maryangela) [inline src/test/java/pacman/WallandDotsTest.java]: identation

File: src/test/java/pacman/WallandDotsTest.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class WallAndDotsTest {
+	@Test
+	public void testExist(){
+        int x=1;
+        int y=1;
+        wall wall1_1 = new wall(x,y, true);
```
Comment (J0c4) [inline src/test/java/pacman/WallandDotsTest.java]: Hi, this is not compiling
image

**Pull request merged**